examples/tutorial/index.ipynb

```diff
@@ -43,38 +43,38 @@
     "## Index and Schedule\n",
     "\n",
     "- *Overview*\n",
-    "   * **15 min** &nbsp;[0 - Setup](./00_Setup.ipynb): Setting up the environment and data files.\n",
-    "   * **30 min** &nbsp;[1 - Workflow Introduction](./01_Workflow_Introduction.ipynb): Overview of solving a simple but complete data-science task, using each of the main PyViz tools.\n",
+    "   * **15 min** &nbsp;[Setup](./01_Setup.ipynb): Setting up the environment and data files.\n",
+    "   * **30 min** &nbsp;[Introductory Example](https://examples.pyviz.org): Overview of solving a simple but complete data-science task, using each of the main PyViz tools.\n",
```

Comment:
I'm not sure what example is meant to be pointed to here, but I don't think it works to just point to all the topic examples; that won't get people started on the material.

Reply:
Yeah I wasn't sure what to do with that. I could point to nyc_taxi?

Reply:
```suggestion
    "   * **30 min** &nbsp;[HoloViz Overview](../talks/Overview.ipynb): Overview of solving a simple but complete data-science task, using each of the main PyViz tools.\n",
```

Reply:
Pointing to the overview seems fine.  Github doesn't seem to let me commit that suggestion; not sure what it's conflicting with.